deployment/network-operator/values.yaml

```diff
@@ -18,6 +18,7 @@
 
 nfd:
   enabled: true
+  nodeFeatureRules: false
```

Comment:
question: where is this being used ?

Reply:
GFD (from the GPU feature discovery) will start using it soon, for this release we have it optional, but for the next one we plan to change it to enabled by default

Reply:
this section is not propagated to node-feature-discovery chart. and currently we dont deploy anything related to GFD via helm.

should this be under node-feature-discovery ? (although i didnt find this parameter in node-feature-discovery chart valuses)

Reply:
we have `enableNodeFeatureApi` under node-feature-discovery section so i guess the parameter here is not needed ?